packages/@aws-cdk/aws-stepfunctions-tasks/lib/sagemaker-create-endpoint-config-task.ts

```diff
@@ -0,0 +1,152 @@
+import * as iam from '@aws-cdk/aws-iam';
+import * as sfn from '@aws-cdk/aws-stepfunctions';
+import { Stack } from '@aws-cdk/core';
+import { getResourceArn } from './resource-arn-suffix';
+import { DataCaptureConfig, ProductionVariants } from './sagemaker-task-base-types';
+
```

Comment:
Switch to using 2 space indent everywhere. From what I can see the rest of the stepfunction-tasks package uses 2-space indent. Follow the same consistency.